packages/ckeditor5-mention/src/mentionui.ts

```diff
@@ -35,54 +37,54 @@ const defaultCommitKeyCodes = [
 
 /**
  * The mention UI feature.
- *
- * @extends module:core/plugin~Plugin
  */
 export default class MentionUI extends Plugin {
+	/**
+	 * The mention view.
+	 */
+	declare private readonly _mentionsView: MentionsView;
```

Comment:
Only observable properties should be `declared`... and special cases like `Command#value`.